README.md

```diff
@@ -1 +1,56 @@
-# portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My Portfolio
+
+> My Portfolio is a simple project build with HTML/CSS only.
+
+
+## Built With
+
+- HTML
+- CSS
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+
+1. Clone this repository or download the Zip folder:
+
+**``git clone https://github.com/iamchristianani/portfolio.git``**
+
+2. Navigate to the location of the folder in your machine:
+
+**``you@your-Pc-name:~$ cd <folder>``**
+
+### Visit And Open Files
+Visit Repo
+[Visit Repo](https://iamchristianani.github.io/portfolio/)
+
+Download Repo
+[Download Repo](https://github.com/iamchristianani/portfolio/archive/refs/heads/main.zip)
+
+
+## Authors
+
+👤 **Author**
+
```

Comment:
- [ ] Please since you are the only contributor to this project you can singularize the `Authors` section to `Author` and replace `Author` with your name. :smile:

Reply:
Author section has been fixed as recommended